src/Console/Shell.php

```diff
@@ -84,6 +84,13 @@ class Shell
     public $OptionParser;
 
     /**
+     * A var for maintain phpversion()
+     *
+     * @var show php version
+     */
+    public $phpVersion;
```

Comment:
I agree, given the usage of this variable it does not require a property at all.